cmd/build/lint.go

```diff
@@ -27,9 +27,17 @@ func (l Lint) goModTidyAll(ctx context.Context) error {
 }
 
 func (Lint) glciFix() error {
-	return shr.Run("golangci-lint", "run", "--fix", "--deadline=15m", "./...", "./apis/...", "./pkg/...")
+	return shr.Run("golangci-lint", "run", "--fix", "--deadline=15m",
+		dotFilepathJoin("..."),
+		dotFilepathJoin("apis", "..."),
+		dotFilepathJoin("pkg", "..."),
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed